backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -181,14 +181,14 @@ object RuntimeAttributesValidation {
     */
   def toRuntimeAttributeDefinition(validation: RuntimeAttributesValidation[_]): RuntimeAttributeDefinition = {
     val name = validation.key
-    val unusedBoolean = false // ... at the time of this writing
+    val attributeRequired = false // ... at the time of this writing
```

Comment:
It's currently unused. I don't know whether this is the right place to comment on that.

If we ever rationalise runtime attribute evaluation, this will need to be filled in.